cdap-proto/src/main/java/co/cask/cdap/proto/metadata/lineage/CollapseType.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.proto.metadata.lineage;
+
+/**
+ * Fields on which lineage relations can be collapsed.
+ */
+public enum CollapseType {
+  ACCESS,
```

Comment:
can we add some docs for these? RUN is pretty clear, but ACCESS and particularly COMPONENT may not be